AdobeStockImageAdminUi/view/adminhtml/web/js/action/authorization.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+define([], function () {
+    'use strict';
+    /**
+     * Consts of response
+     *
+     * RESPONSE_PATTERN - pattern of response
+     * RESPONSE_CODE_INDEX index of response code
+     * RESPONSE_MESSAGE_INDEX index of response message
+     * RESPONSE_SUCCESS_CODE success code
+     */
+    var RESPONSE_PATTERN = /auth\[code=(success|error);message=(.+)\]/,
```

Comment:
these constants should be passed from backend as they should match backend response, let's move the to authConfig as well and set on backend based on Callback controller constants.